### DISCUSSION TITLE
[chart]: include group/series label in Histogram tooltip

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#18424] [Ideas]
currently, group label is not showing in the tooltip when user hovering on a Histogram with bars split by multiple groups/series. 

Proposed enhancement: 
<img width="1781" alt="Screen Shot 2020-12-16 at 1 28 08 PM" src="https://user-images.githubusercontent.com/67837651/102410589-9a8cbb00-3fa5-11eb-88c6-cbf4465be145.png">

**Comment by rusackas** [comment]
Closing this, and carrying it over as a comment on https://github.com/apache/superset/discussions/24215
